Error building flasher-tool

Trying to run customizing the iso offline

```zsh
$ go version
go version go1.25.4 darwin/arm64
```

```zsh
$ go install github.com/lxc/incus-os/incus-osd/cmd/flasher-tool@latest
# github.com/lxc/incus-os/incus-osd/internal/storage
../../../go/pkg/mod/github.com/lxc/incus-os/incus-osd@v0.0.0-20251201155339-f37f4ddc4946/internal/storage/storage.go:105:23: cannot use s.Dev (variable of type int32) as uint64 value in argument to unix.Major
../../../go/pkg/mod/github.com/lxc/incus-os/incus-osd@v0.0.0-20251201155339-f37f4ddc4946/internal/storage/storage.go:106:23: cannot use s.Dev (variable of type int32) as uint64 value in argument to unix.Minor
../../../go/pkg/mod/github.com/lxc/incus-os/incus-osd@v0.0.0-20251201155339-f37f4ddc4946/internal/storage/storage.go:181:17: invalid operation: s.Bsize * int64(s.Bfree) (mismatched types uint32 and int64)
# github.com/lxc/incus-os/incus-osd/internal/seed
../../../go/pkg/mod/github.com/lxc/incus-os/incus-osd@v0.0.0-20251201155339-f37f4ddc4946/internal/seed/seed.go:53:64: too many arguments in call to unix.Mount
        have (string, string, string, number, string)
        want (string, string, int, unsafe.Pointer)
../../../go/pkg/mod/github.com/lxc/incus-os/incus-osd@v0.0.0-20251201155339-f37f4ddc4946/internal/seed/seed.go:56:68: too many arguments in call to unix.Mount
        have (string, string, string, number, string)
        want (string, string, int, unsafe.Pointer)
../../../go/pkg/mod/github.com/lxc/incus-os/incus-osd@v0.0.0-20251201155339-f37f4ddc4946/internal/seed/seed.go:147:51: too many arguments in call to unix.Mount
        have (string, string, string, number, string)
        want (string, string, int, unsafe.Pointer)
../../../go/pkg/mod/github.com/lxc/incus-os/incus-osd@v0.0.0-20251201155339-f37f4ddc4946/internal/seed/seed.go:150:55: too many arguments in call to unix.Mount
        have (string, string, string, number, string)
        want (string, string, int, unsafe.Pointer)
# github.com/lxc/incus-os/incus-osd/internal/secureboot
../../../go/pkg/mod/github.com/lxc/incus-os/incus-osd@v0.0.0-20251201155339-f37f4ddc4946/internal/secureboot/secureboot.go:241:55: too many arguments in call to unix.Mount
        have (string, string, string, number, string)
        want (string, string, int, unsafe.Pointer)
```

--- a/doc/getting-started/download.md
+++ b/doc/getting-started/download.md
@@ -30,18 +30,22 @@ image that's ready for immediate use.
 
 ## Flasher tool
 
-The flasher tool is provided for more advanced users who need
+The flasher tool is provided for advanced users who need
 to perform more customizations of the install seed than the web-based customizer
 supports.
+
+```{note}
+Currently, the flasher tool is only available for Linux systems.
+```
 
 It can be built and run on a system with the Go compiler installed using:
 
     go install github.com/lxc/incus-os/incus-osd/cmd/flasher-tool@latest
     flasher-tool
 
-when run, you will first be prompted for the image format you want to use, either ISO
-(default) or raw disk image. Note that the ISO isn't a hybrid image; if you
-want to boot from a USB stick you should choose the raw disk image format.
+When run, you will first be prompted for the image format you want to use, either `iso`
+(default) or an `img` (raw) disk image. Note that the ISO isn't a hybrid image; if you
+want to boot from a USB stick you should choose the `img` disk image format.
 
 The flasher tool will then connect to the Linux Containers CDN and download the
 latest release.
@@ -76,3 +80,6 @@ preseed:
 
 After writing the image and exiting, you can then install IncusOS from the
 resulting image.
+
+A variety of command line arguments can be provided to the flasher tool,
+allowing for its use in an automated fashion such as from within a script.

